pkg/plugin/api/server.go

```diff
@@ -312,7 +325,7 @@ func (c *grpcServer) Create(ctx context.Context, in *proto.CreateRequest) (*prot
 	return &proto.CreateResponse{}, nil
 }
 
-// Get gets an object.
+// Delete gets an object.
```

Comment:
Super minor nitpick, might want to update this comment to `Delete deletes an object` to avoid potential confusion